data/systemd/snapd.mounts.target

```diff
@@ -0,0 +1,4 @@
+[Unit]
+Description=Mounted snaps
+Before=local-fs.target
```

Comment:
this moves all the mounts much earlier if I understand the systemd dep chain? I think it's fine but double checking

Reply:
All `.mount` units that have default dependencies should have `Before=local-fs.target` implicitly. So that should not push the mounts before.

Reply:
See https://www.freedesktop.org/software/systemd/man/systemd.mount.html#Default%20Dependencies

Reply:
yes, sorry, the actual chaining is done now by the Wants=snapd.mounts.target and After=snapd.mounts.target in snapd.service